### DISCUSSION TITLE
Compatibility with fetch-mock

### DISCUSSION THREAD

**Discussion by josh08h** [sindresorhus/ky#247] [Q&A] [answered]
Is this package compatible with fetch-mock (https://github.com/wheresrhys/fetch-mock)?

**Comment by sindresorhus** [comment]
`ky` uses `fetch` underneath, so if `fetch-mock` handles `fetch` according to the standard, then yes. You might need to disable retrying though.

**Comment by sholladay** [comment, marked as answer]
Yes they work fine together. We use this setup at my company.

Are you having any specific issues?

**Comment by sholladay** [comment]
Closing since the question was answered. There's nothing special about using `fetch-mock` with Ky. Just use it as instructed in their documentation. For further support, post a detailed question on Stack Overflow.

Happy to re-open if you think there's a bug in Ky related to using `fetch-mock`.
